### DISCUSSION TITLE
SFTP

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#10163] [General]
It would be fantastic to ftp file on save to remote dev server. I can't work remotely with helix because I don't want a remote '.git' folder. 

I tried using fswatch and rsync but there is an annoying delay with this approach.

**Comment by lmerz1** [comment]
This (unfortunately) sounds like a prime example of "plugin functionality", aka a topic for a future date.
I used to use [this extension](https://marketplace.visualstudio.com/items?itemName=Natizyskunk.sftp) in VSCodium, for comparison.

**Reply by ghost** [reply]
Thanks for your reply. In desperation I added this to config.toml: 

C-t = [
    ":write",
    ":sh ./my_script",
]
 
Might be useful to others reading the post.
